src/partials/footer.html

```diff
@@ -30,7 +30,6 @@ <h3 class="footer__title-second">with passion </h3>
         <div class="copyright">
             <a class="copyright__link" href="#">
             <div> © 2019 - IceCream. All Rights Reserved. </div>
```

Comment:
A ten div tu chyba nie potrzebny skoro to ma być .copyright__link
to może być tekst między tagami linku